store.html

```diff
@@ -0,0 +1,157 @@
+
+                    
+                    <!-- for Accessibility -->
+
+                    <fieldset class="hide-this">
+                        <legend class='hide-this'>Your Name</legend>
+
+                        <div class='form-flex show-this'>
+
+                            <div class='width-475'>
+                            <label class='label' for="first-name">First name *</label>
+                            <input class='input' id='first-name' type='text' required minlength=2>
+                            </div>
+
+                            <div class='width-475'>
+                            <label class='label' for="last-name">Last name *</label>
+                            <input class='input' id='last-name' type='text' required minlength=2>
+                            </div>
+
+                        </div> 
+                    </fieldset>
+
+                    <fieldset class='hide-this'>
+                        <legend class='hide-this'>Your Address</legend>
+
+                        <div class='your-address show-this'>
+
+                            <label class='label' for='address-one'>Address *</label>
+                            <input class='input' type="text" id="address-one" required minlength=4>
+
+                            <label class='label' for='address-two'>Address 2</label>
+                            <input class='input' type="text" id="address-two">
+
+                            <div class='form-flex'>
+
+                                <div class='width-575'>
+                                    <label class='label' for='city'>City *</label>
+                                    <select class='select input' required id='city' form='order-form'>
+                                        <option value="" disabled selected>Select your city...</option>
+                                        <option value='london'>London</option>
+                                        <option value='budapest'>Budapest</option>
+                                        <option value='new-york'>New York</option>
+                                    </select>
+                                </div>
+                                
+                                <div class='width-375'>
+                                    <label class='label' for='postcode'>Postcode *</label>
+                                    <input type="text" class="input" id="postcode" required minlength=4 maxlength=8>
+                                </div>
+
+                            </div>
+
+                        </div>
+                    </fieldset>
+
+                    <fieldset class='hide-this margin-top-15'>
+                        <legend class='margin-bottom-1 show-this'>Select a colour</legend>
+
+                        <div class='show-this'>
+
+                            <input class='store-radio' type="radio" required value='karma-orange' name='colour' id='karma-orange'>
```

Comment:
This is default form option, so it should be set as 'checked' 
```suggestion                            
                            <input class="store-radio" type="radio" required="" value="karma-orange" name="colour" id="karma-orange" checked>
```